apps/wallet/src/ui/app/pages/swap/QuoteAssets.tsx

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { coinsMap } from '_app/hooks/useDeepBook';
+import Overlay from '_components/overlay';
+import { useActiveAddress, useCoinsReFetchingConfig } from '_hooks';
+import { TokenRow } from '_pages/home/tokens/TokensDetails';
+import { useSuiClientQuery } from '@mysten/dapp-kit';
+import { useSearchParams } from 'react-router-dom';
+
+const recognizedCoins = Object.values(coinsMap);
+
+function QuoteAsset({
+	coinType,
+	borderBottom,
```

Comment:
I never like props like this, generally there's better ways to do it through the parent component instead of adding one-off list display logic here. I know this is inherited from `TokenRow`, so not urgent to fix, just something we really _should_ fix at some point (maybe just through the process of the refactor)